### ISSUE TITLE
Copy Link Text

### ISSUE THREAD

**Issue by NokTham** [dhruvtv/copylinkaddress#5]
Good afternoon, I was looking for an easy way of copying the link text without all the trouble that it involves, and I found your extension, which does a great job, but for links. Is it possible to implement copying the link text?

**Comment by SmartManoj** [comment]
https://github.com/dhruvtv/copylinkaddress/blob/24d78bd0e4ae06a5996ad7f12e7924dcf406cf52/copylinkaddress.js#L84

Have to use $(this).text()

Bind with Ctrl + Alt + c

**Comment by dhruvtv** [comment]
Thanks for raising this!

I wanted to keep the extension super simple and focus on the usability pain with right-clicking and selecting 'Copy Link Address'.

About copying link text, do you find that selecting the link text and hitting Cmd-C to be very hard?

I'm not opposed to implementing this, but we need to find a good key combination for both macOS and Windows. Cmd-Option-C is already taken.

**Comment by eugenesvk** [comment]
> selecting the link text and hitting Cmd-C to be very hard?

Yes, when you need to do that precisely and not forget the first / last letter, it's not like you have double click available to you to select the whole word or shift+ctrl+arrow to select by word on the web!

(meanwhile I'm using Alt+C as a secondary shortcut)

> but we need to find a good key combination for both macOS and Windows. Cmd-Option-C is already taken

Ideally this should be a customizable shortcut

Another alternative is double tap of Ctrl+C,C within X seconds (also should be custom) to copy text while Ctlr+C to copy address
